mdf_forge/forge.py

```diff
@@ -437,6 +437,60 @@ def match_titles(self, titles):
         return self
 
 
+    def match_years(self, years=[], min = None, max = None, include=True):
+        """Add years and limits to the query.
+
+        Arguments:
+        years   (int, str or list of int or str): The years to match. Default [].
+        min     (int of str): The lower range of years to match..
+        max     (int of str): The upper range of years to match.
+        include (bool): If True, will add min and max years to the range. If False, they will be excluded.
+                          Default True.
+        limit (int): The maximum number of results to return.
+                     The max for this argument is the SEARCH_LIMIT imposed by Globus Search.
+        info (bool): If False, search will return a list of the results.
+                     If True, search will return a tuple containing the results list,
+                        and other information about the query. Default False.
+        Returns:
+        self (Forge): For chaining.
+        """
+        if not years and not min and not max:
```

Comment:
We should check explicitly for the arguments being `None`, because `0` could be a value the user enters.

Reply:
Done